lookout/core/data_requests.py

```diff
@@ -56,6 +68,31 @@ def get_bblfsh(self) -> bblfsh.aliases.ProtocolServiceStub:
                 bblfsh.aliases.ProtocolServiceStub(self._get_channel())
         return stub
 
+    def check_bblfsh_driver_versions(self, versions: Iterable[str]):
+        """
+        Ensure that the Babelfish drivers match the required versions.
+
+        The check is performed by `packaging.version`.
+
+        :param versions: setup.py-like version specifiers, e.g. "javascript==1.3.0".
+        :return: Nothing
+        :raise UnsatisfiedDriverVersionError: if there is one or more mismatches.
+        """
+        existing = self.get_bblfsh().SupportedLanguages(
+            bblfsh.aliases.SupportedLanguagesRequest()).languages
+        mismatched = []
+        for reqstr in versions:
```

Comment:
I think we do not use types in names
How about `requirement`?
```suggestion
        for requirement in versions:
```

Reply:
This is fine